restcomm/restcomm.testsuite/pom.xml

```diff
@@ -273,18 +273,18 @@
             <scope>test</scope>
         </dependency>
 
-		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-all</artifactId>
-			<version>1.8.5</version>
-			<scope>test</scope>
-		</dependency>	
-		<!-- <dependency>
-  			<groupId>org.powermock</groupId>
-  			<artifactId>powermock-module-testng-agent</artifactId>
-  			<version>1.6.6</version>
-  			<scope>test</scope>
-		</dependency> -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.8.5</version>
+            <scope>test</scope>
+        </dependency>	
+        <!-- <dependency>
```

Comment:
i believe we can remove the portion of code that is note needed instead of keeping it in comments, i know its coming from master and not this branch, but just a suggestion since its appearing on diff.

Reply:
as you mentioned code is full of commented code, but i can remove this one in particular